dev/benchmarks/k6LogBatchPerfScript.js

```diff
@@ -0,0 +1,90 @@
+import http from "k6/http";
+
+export default function () {
+  // Set base url from environment variable, the variable ' -e HOSTNAME = xxxx ' must be added to command line argument
+  const base_url = "http://" + __ENV.HOSTNAME + "/api/2.0/mlflow";
```

Comment:
we probably want to use `MLFLOW_TRACKING_URI` instead, and check that it starts with `http/https`